### PR TITLE
Splash Screen

### PR DESCRIPTION
**Pull request by jmonto55**
This PR includes the frontend for the splash screen with log in and sign up buttons.

img

### CODE REVIEW

File: src/components/Splash.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    mediaQuery1.addEventListener('change', handleMediaQueryChange);
+
+    return () => {
+      mediaQuery1.removeEventListener('change', handleMediaQueryChange);
+    };
+  }, []);
+
+  return (
+    <div className="w-screen h-screen bg-center bg-no-repeat" style={{ backgroundImage: `url(${houseImage})` }}>
```
Comment (SimonGideon) [inline src/components/Splash.jsx:29]: I'll suggest you make the buttons a little bigger since its not visible enough with the background image.

**Review by SimonGideon — CHANGES_REQUESTED** [review]
## Hi @jmonto55 , 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Splash.jsx (hunk 1 above)
Comment (jmonto55) [reply src/components/Splash.jsx:29]: You're right, teammate, I will make those buttons bigger to increase accessibility. 🔧

**Review by SimonGideon — APPROVED** [review]
## STATUS: APPROVED 🎉🎉🎉

Hi @jmonto55 ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approve-image

## Highlights
✔ Splendid work
✔ Great design and use of the chosen framework
✔ Good code design and implementation

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------

**Pull request merged**